keps/prod-readiness/sig-scheduling/4247.yaml

```diff
@@ -1,3 +1,5 @@
 kep-number: 4247
 beta:
   approver: "@wojtek-t"
+stable:
+  approver: "@wojtek-t"
```

Comment:
/assign @wojtek-t 

Please assign someone else if needed.

Reply:
it's fine - feel free to put my on other sig scheduling keps too.